### DISCUSSION TITLE
The error message string does not whow on search results, why? thanks

### DISCUSSION THREAD

**Discussion by shivah23** [x64dbg/x64dbg#3056] [Q&A]
hello, thanks for the x64dbg im trying to learn.

i just want to ask what to do if the string from error message that im supposed to trace  were not showing on results from All Modules (string) search ?

**Comment by torusrxxx** [comment]
You need to improve your skills. Although string search is the easiest tool, it isn't guaranteed to work because the string may be dynamically assembled, encrypted, loaded from external resources, etc.
